third_party/proto/kythe/README.md

```diff
@@ -0,0 +1,5 @@
+# Kythe proto schema
+
+Proto schemas imported from https://github.com/kythe/kythe
+
+*   https://github.com/kythe/kythe/blob/master/kythe/proto/storage.proto
```

Comment:
Should we add a link to the exact git-hash revision or Kythe release we took the current file version ? Might be useful for later updates (though I suspect the proto changes rarely at this point).

Reply:
Yep. Done.